components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.publishevent.ui/src/main/java/org/wso2/carbon/mediator/publishevent/ui/PublishEventMediator.java

```diff
@@ -137,6 +157,13 @@ public OMElement serialize(OMElement parent) {
 	 * @param elem OMElement to be converted to publishEvent Mediator Object.
 	 */
 	public void build(OMElement elem) {
+		String async = elem.getAttributeValue(ASYNC_Q);
+		this.setIsAsync(async);
+
+		if(Boolean.parseBoolean(async)){
```

Comment:
Minor formatting issue, should be if (...)

Reply:
fixed with e0b1599